config/prod/s3-synapse-sync.yaml

```diff
@@ -13,8 +13,9 @@ parameters:
   BucketVariables: >-
     {
       "htan-dcc-center-a":{"SynapseProjectId":"syn22343754"},
-      "htan-dcc-bu":{"SynapseProjectId":"syn22124336"},
-      "htan-dcc-ohsu":{"SynapseProjectId":"syn22093319"}
+      "htan-dcc-ohsu":{"SynapseProjectId":"syn22093319"},
+      "htan-dcc-tma-tnp":{"SynapseProjectId":"syn22041595"},
+      "htan-dcc-msk":{"SynapseProjectId":"syn23448901"}
```

Comment:
don't you want to add vanderbilt bucket to this list?

Reply:
Not at the moment. vanderbilt will upload data via Synapse, and we just plan to attach this bucket to their project as an external storage location (not set up with the sync), like the CHOP bucket Xengie provisioned last week.